pom.xml

```diff
@@ -176,9 +176,10 @@
                         <exclude>.github/**</exclude>
                         <exclude>src/test/resources/certs/*</exclude>
                         <exclude>src/test/**/*.log</exclude>
-                        <exclude>src/test/resources/META-INF/service/*</exclude>
+                        <exclude>**/src/main/resources/META-INF/services/*</exclude>
```

Comment:
why did you add prefix placeholder before the root directory /src?

Reply:
I tried it. When running `mvn -B clean apache-rat:check` and building RocketMQ Spring Boot, the rocketmq-spring-boot-samples project will also be checked. If I do not add the prefix placeholder, an unapproved license error will be reported. Like this: [https://travis-ci.org/github/apache/rocketmq-spring/builds/748732956](https://travis-ci.org/github/apache/rocketmq-spring/builds/748732956)